composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.6",
         "apishka/cs": "~1.0.0",
-        "composer-plugin-api": "~1.0.0",
+        "composer-plugin-api": "^1.0.0",
```

Comment:
~1.1.0 would fail as currently the plugin api version is 1.0.0, and I didn't want to change more than needed.